src/lightning_app/utilities/cloud.py

```diff
@@ -11,14 +11,16 @@
 
 def _get_project(client: LightningClient, project_id: str = LIGHTNING_CLOUD_PROJECT_ID) -> V1Membership:
     """Get a project membership for the user from the backend."""
+    print(client.projects_service_list_memberships)
+    assert False
```

Comment:
could be removed ?

Reply:
I'm still debugging. The ****** CI won't let this test pass. The mock patch does not work in the CI but it works locally.